cmd/cluster-agent-cloudfoundry/app/app.go

```diff
@@ -201,6 +202,14 @@ func run(cmd *cobra.Command, args []string) error {
 		return log.Errorf("Error while starting agent API, exiting: %v", err)
 	}
 
+	if config.Datadog.GetBool("cluster_agent.admission_controller.enabled") {
+		log.Info("Starting admission controller server")
+
+		if err = admissioncontroller.StartServer(); err != nil {
+			return log.Errorf("Error while starting admission controller server, exiting: %v", err)
+		}
+	}
+
```

Comment:
In Go it's uncommon to have so many empty lines. If you look at most Go libraries (especially the language repository `golang/go`) you will notice that empty lines are rarely used in code. Especially around curly braces which already provide enough spacing.